src/components/pod/pod.spec.js

```diff
@@ -138,26 +141,23 @@ describe("Pod", () => {
     });
 
     it("onDelete prop function gets invoked by clicking delete action button", () => {
```

Comment:
I _think_ a lot of these test cases don't match the description provided to the `it` block, would you mind reviewing them and updating if needs be?